test/integration/verify.go

```diff
@@ -418,3 +420,56 @@ func verifyNoP4RuntimeEntries(t *testing.T, expectedValues p4RtValues) {
 			fmt.Sprintf("%v should not contain any entries", table))
 	}
 }
+
+func verifyBessEntries(t *testing.T, bess *fake_bess.FakeBESS, testdata *pfcpSessionData, expectedValues p4RtValues, ueState UEState) {
```

Comment:
I suggest creating two files: `verify_up4.go` and `verify_bess.go`. Then, we can move verify methods for BESS to `verify_bess.go`.